### DISCUSSION TITLE
tango面向的目标用户是哪些群体？

### DISCUSSION THREAD

**Discussion by passby6someone** [NetEase/tango#115] [General]
刚刚体验了一下，想了解下预想的使用场景😂

如果是面向后端开发提供工具快速开发简单的中后台系统，感觉暴露出来的设置项太多了，后端也不关心前端源码具体什么样。

如果是面向前端开发提供的开发工具，props选项感觉有点繁琐，感觉直接写会快一点，变量和接口的设置感觉会有一定的学习成本。

试了能改源码然后反应到设置项里体验很好，没有DSL渲染器黑盒这一点非常好评！

但是如果是简单布局的页面拖拖拽拽可能比直接写没有太多提效，如果是有复杂多选项联动和多功能表格的页面，好像也承载不了。。。

想了解下是出于什么思路设计的这个低代码平台😂

**Comment by wwsun** [comment]
开源版本只包括了最核心的搭建引擎部分。你说的大部分问题都指向了**物料的封装程度和易用性**上，本身和搭建引擎的能力并不相关。方案思路上，建议了解下我们之前的分享。

对于**低代码物料部分**，因为工作量较大，目前并不在我们的开源范围。

**Comment by ZKK666** [comment]
目前对于tango这种低代码平台对于前端开发工程师的提效是否可观确实存在一些疑虑。简单的页面其实很快可以自己写完，复杂的页面或者个性化的页面，低代码或许不好支持。低代码对于前端工程师的提效这条路不好走。当前如果是非前端人员的话，这种低代码平台提效应该还是很明显的，减少了前端知识的学习成本。
